datagenerator/src/main/java/com/linkedin/restli/datagenerator/csharp/CSharpRythmTransformer.java

```diff
@@ -45,6 +47,14 @@
     typeNameMap.put(DataSchema.Type.LONG, "long");
   }
 
+  private static final String PATH_ROOT_DIRECTORY = "com";
+  private static final Set<String> pathRootSet = new HashSet<>();
+  static
+  {
+    pathRootSet.add("/" + PATH_ROOT_DIRECTORY + "/"); // Unix delimiters
```

Comment:
Should we use Java File.separator instead of hard-code delimiters for Unix and Windows?